doc/reference/eventengine.rst

```diff
@@ -38,6 +38,117 @@ of alerts. The default configuration looks somewhat like this:
+         severity: 5
+
+       - netbox.organization.id: foobar
+         severity: '+2'
+
+This configuration starts off by assigning a *default severity* level of **3**
+to every alert that :program:`Event Engine` generates, regardless of what the
+original severity value of the event was.
+
+Then follows a list of rules that will be processed *in the order they appear*
+in the file. Each rule consists of:
+
+- One or more alert attribute match expressions.
+- One severity value modification expression to be applied to an alert that
+  matches the attribute expressions.
+- Optionally, a sub-list of more rules to further apply to any alert that
+  matched the expressions of this rule.
+
+The first example rule will match any alert whose ``alert_type`` value equals
+``boxDown`` (NAV's alert type for a lasting "box is unreachable" incident). Any
+such alert will be assigned a severity level of **2**.  Furthermore, the rule
+lists two additional sub-rules to ensure that if the ``boxDown`` alert was
+issued for any netbox (IP Device) whose category is a router (a category id of
+either ``GSW`` or ``GW``), the severity is set to the most critical level of
+**1**.
+
+The second top-level example rule will match any alert whose type is
+``boxDownWarning``, and set its severity to the least critical level of
+**5**. This is the stateless early warning the :program:`Event Engine` issues a
+few minutes before declaring a stateful ``boxDown``. It is safe to consider
+this type of alert as only *informational*.
+
+The final top-level example rule will match any alert whose associated netbox
+(IP Device) is owned by the organizational id ``foobar``. This rule uses a
+*severity modifier expression* of ``+2``, which will add ``2`` to the current
+alert's existing severity value.
+
+In summary, if a ``boxDown`` alert is dispatched for a router in your network,
+this rule set will ensure its severity is set to **1**. However, if the router
+belongs to your less important ``foobar`` department, two severity levels will
+be deducted, and the alert comes out with a severity of **3**.
+
+
+Modifier expressions
+~~~~~~~~~~~~~~~~~~~~
+
+There are three types of supported severity modifier expressions for use in rules:
+
+1. Absolute values: An integer in the range 1-5 will *replace* a matching
+   alert's current severity level with this integer.
+2. Addition: Prefixing an integer with ``+`` will add this value to the
+   matching alert's existing severity value. Remember that higher numbers are
+   considered *less* severe.
+3. Subtraction: Prefixing an integer with ``-`` will subtract this value from
+   the matching alert's existing severity value. Remember that lower numbers
+   are considered *more* severe.
+
+You need not worry that using addition or subtraction modifiers will result in
+an invalid severity outside the range of 1-5. Any attempt to assign a value
+outside the legal range will only result in a value at one of the range's
+endpoints.
```

Comment:
High verbosity level may not be needed here.

I suggest condensing it a bit:

```
1. Absolute values: An integer in the range 1-5 will *replace* a matching
   alert's current severity level.
2. Relative values: Prefixing an integer with ``+`` (or ``-``) will *increase* (or decrease)
   the existing severity value by the given value. NAV will ensure that this operation does
   not exceed the valid range of 1-5.
```

Reply:
I like this, thank you! :)